### PR TITLE
fix: Improve Scope of Table Name Regex to Resolve Errors

### PR DESCRIPTION
**Pull request by darunrs**
The previous regex was not able to support DDL where the table names were not encapsulated in quotes. Through more research, I learned as well that putting the table name in quotes causes the table name it be case sensitive and also allow special characters. I also omitted checking for "IF NOT EXISTS". So, I updated the regex to allow optional white space and "IF NOT EXISTS". Then, I skip a SINGLE optional double quote, match anything until another optional double quote or white space. Everything between the white space following either TABLE or EXISTS and potentially two double quotes is now the table name. The name can have special characters so I ensure that any such names have double quotes added back in. I also support table names defined by schema.tableName. Finally, for testing I did the following: 

Tested generating table names in front end by opening all indexers and running debug mode to generate methods. All indexers succeeded, including notable ones like bos_quests, social_feed, and widget_testing. 

I also added a stress test schema which is a valid DDL schema (verified using format button) which contains all kinds of edge cases, and ensure the generated method names correctly sanitize the special characters to underscores. There might be other ways to more cleanly implement this transition of special characters to function name but I think this is fine for now, and we hopefully will have autocomplete for the context object soon. 

Finally, I also did an integration test using the existing insert function to confirm functionality has remained unimpacted.

### CODE REVIEW

**Review by morgsmccauley — COMMENTED** [review]
Mostly looks good, but for now I think we should remove the errors that are thrown when trying to create `context.db`. Instead we should just log the errors and not add it to the context. This avoids us breaking existing indexers unintentionally. We can be more strict once we have the final solution.

File: runner/src/indexer/indexer.test.ts [hunk 1]
```diff
@@ -62,7 +62,99 @@ describe('Indexer unit tests', () => {
         "block_timestamp" DECIMAL(20, 0) NOT NULL,
         "receipt_id" VARCHAR NOT NULL,
         CONSTRAINT "post_likes_pkey" PRIMARY KEY ("post_id", "account_id")
-      );'`;
+      );`;
+
+  const STRESS_TEST_SCHEMA = `
+CREATE TABLE
+  creator_quest (
```
[the remark this answers is not in the document]
Comment (darunrs) [reply runner/src/indexer/indexer.test.ts]: This was a great catch. Turns out the regex would have \s+ twice. So, the space between TABLE and the name would only satisfy one and it would be skipped.

**Review by morgsmccauley — APPROVED** [review]
